lib/test/cdo/test_log_object.rb

```diff
@@ -0,0 +1,41 @@
+require_relative '../test_helper'
+require 'cdo/log_object'
+
+class LogObjectTest < Minitest::Test
```

Comment:
Is it redundant to include Object in the class name?  (Similar for instance name, which includes _object.)